scene/resources/animation.cpp

```diff
@@ -427,6 +427,13 @@ bool Animation::_set(const StringName &p_name, const Variant &p_value) {
 		} else {
 			return false;
 		}
+	} else if (prop_name == "loop"){
+		if (p_value) {
+			loop_mode = Animation::LoopMode::LOOP_LINEAR;
+		}else {
```

Comment:
```suggestion
		} else {
```
Please follow the code style, see [here](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html)